docs/internals/howto-release-django.txt

```diff
@@ -414,6 +414,8 @@ Now you're ready to actually put the release out there. To do this:
 #. Add a link to the blog post in the topic of the ``#django`` IRC channel:
    ``/msg chanserv TOPIC #django new topic goes here``.
 
+.. _django-forum: https://forum.djangoproject.com
```

Comment:
In this case link is unnecessary.

Reply:
Yes you are correct, i deleted this line in my recent commits